src/gui/src/viewer/zoom-window.cpp

```diff
@@ -224,9 +244,11 @@ ZoomWindow::~ZoomWindow()
 	m_detailsWindow->deleteLater();
 
 	// Fix for weird Linux crash (issue #2190)
-	m_stackedWidget->removeWidget(m_videoWidget);
-	m_videoWidget->setParent(nullptr);
-	m_videoWidget->deleteLater();
+	if (m_videoWidget != nullptr) {
+		m_stackedWidget->removeWidget(m_videoWidget);
+		m_videoWidget->setParent(nullptr);
+		m_videoWidget->deleteLater();
+	}
```

Comment:
This is not in my file. I don't know why it's here.

Reply:
This happens when you rebase/merge after starting your change.
This is a change I made after you created your PR but before you published it I guess 😄
It will disappear if you rebase (I rebased for you so it actually already disappeared)